File: templates/matlab/observer_estimation_before_after.m
function fig = observer_estimation_before_after()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('slope', 1720, 'observer and state estimation: before-after slope', 'observer and state estimation', 'before-after slope');
end

File: templates/python/observer_estimation_before_after.py
"""observer_estimation_before_after: 观测器与状态估计前后斜率对比（slope 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='observer and state estimation: before-after slope'):
    return make_template_figure('slope', seed=1720, title=title, domain='observer and state estimation', topic='before-after slope')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)
